### ISSUE TITLE
Can't Install python-binance On Latest Python Version

### ISSUE THREAD

**Issue by oriamrm** [sammchardy/python-binance#515]
Installation fails every time, printing a vague error message. Tried installing from within PyCharm and also from the Windows CMD. Both pip versions are up to date (11.5.2020).

Environment:
 - Python version: 3.8.2 (Current Latest)
 - Virtual Env: virtualenv
 - OS: Windows 10
 - python-binance version

Logs are appended as files.
[cmd_output.txt](https://github.com/sammchardy/python-binance/files/4610551/cmd_output.txt)
[pip_logfile.txt](https://github.com/sammchardy/python-binance/files/4610552/pip_logfile.txt)

And I just want to say, I have been using python-binance & python-kucoin for a long time. Thank you Sam, you make my day to day work a lot easier :)

**Comment by oliver-zehentleitner** [comment]
The error is "since package 'wheel' is not installed."

On Windows you can do that with `py -m pip install wheel --upgrade`.

**Comment by oriamrm** [comment]
Thanks for pointing that out. I did see it before, and I've made sure my wheel is up to date. So this time just to be sure I reinstalled it, and then another error message regarding Visual C++ packages was shown before the aforementioned error:
`error: Microsoft Visual C++ 14.0 is required. Get it with "Build Tools for Visual Studio": https://visualstudio.microsoft.com/downloads/`

For anyone viewing this thread in the future: You need the C++ package**s**, as in plural. Easiest way is to grab the Visual Studio 20__ installer, then select   Desktop Development With C++ and install just that.

TL;DR Problem solved, thanks.
